packages/host/app/resources/search.ts

```diff
@@ -203,6 +190,9 @@ export class SearchResource extends Resource<Args> {
         this._instances.length,
         ...results.map((instance) => instance),
       );
+      if (this.#doWhileRefreshing) {
+        this.#doWhileRefreshing();
+      }
```

Comment:
This is a nice pattern I found that enables me to set the `activeSpec` in local storage when a search is returned. Then I handle the `activeSpec` independently with a card resource